### ISSUE TITLE
[Function _n] Bug when arguments are php parameters!

### ISSUE THREAD

**Issue by better-studio** [WordPress/theme-check#180]
Paste following code in any theme ( 'twentyfifteen' in this example):

```php
$t_min = __( '%s min', 'twentyfifteen' );

$t_mins = __( '%s minuets', 'twentyfifteen' );

$mins = 3;

$since = sprintf( _n( $t_min, $t_mins, $mins, 'twentyfifteen' ), $mins );
```

Theme check returns this warning but of you check the code is correct:
WARNING: Found a translation function that is missing a text-domain. Function _n, with the arguments 'twentyfifteen'

But when you write code with strings inside _n and without params it will be ok!

```php
$mins = 3;

$since = sprintf( _n( '%s min', '%s minuets', $mins, 'twentyfifteen' ), $mins );
```

Ok i know we should put strings inside _n but that strings are coming from out Theme Translation Panel and there is no way for us!

**Comment by grappler** [comment]
This code will not work.

`$since = sprintf( _n( $t_min, $t_mins, $mins, 'twentyfifteen' ), $mins );`

 If the site is in one language then the text can be saved in that language in the database. If the site is multilingual then you need a extra plugin for this.

**Comment by riddbengkok** [comment]
'_n' => array('singular', 'plural', 'domain')
just give 3 argument bro , i have try and it works
